README.rst

```diff
@@ -38,13 +38,14 @@ also you can install VmShepherd manually using a make:
 
   make install
 
-Also we provide a dockerfile which can be used for a development installation:
+There is also a third way with using a docker container:
```

Comment:
I'm not so sure that we want to introduce THIRD way to run this. Even 2 may be to much imho